### DISCUSSION TITLE
Add search query to URL query params

### DISCUSSION THREAD

**Discussion by babakfp** [tailwindlabs/heroicons#1089] [Ideas]
It would be nice if we could search icons in https://heroicons.com and later share the URL with someone and they see the icons that we searched for.

By the way, thanks for the hard work behind all TailwindLabs products/projects.

**Comment by LarsFlieger** [comment]
You can anchor link icons with `#ICON_NAME-btn`

**Example**
Icon name: `cog`
https://heroicons.com/#cog-btn

It scrolls a bit of because of the sticky navbar but maybe it helps finding it.
